README.md

```diff
@@ -53,6 +53,16 @@ Edit the .env file and set the database password
 docker compose up
 ```
 
+3. Run the migrations
+```sh
+yarn prisma migrate dev
+```
+
+4. Run the seeds
+```sh
+yarn prisma db seed
+```
+
```

Comment:
Docker compose does migrations and seeding this internally so you don't need to run it again as part of the Database setup.

Can we instead make a section like so
```suggestion
### Database and ORM commands

Re-run migrations with `yarn prisma migrate dev`

Create a new migration by:
1. Editing schema.prisma and save
2. Run `yarn prisma migrate  dev --name snake_case_title`
3. Before running the backend, regenerate the Prisma Client with `yarn prisma generate`

```

Sadly I can't do triple-backticks in the suggestion section or Markdown has a parsing fit, but feel free to adjust further if you like.

Lastly, seeding shouldn't be something we regularly need to run so probably fine to skip here as it's dockerized.